galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -644,6 +645,7 @@ def configure_dynamic_settings(settings: Dynaconf) -> Dict[str, Any]:
     change the value before it is returned allowing reading overrides from
     database and cache.
     """
+    print(f'configure_dynamic_settings ... {settings.get("GALAXY_DYNAMIC_SETTINGS")}')
```

Comment:
we can remove the prints?

Reply:
Yeah, once we're sure this is the path forward architecturally